src/Lattice/CrystalLattice.h

```diff
@@ -68,8 +68,8 @@ struct PosUnit
  *expression template operations with variable-cell algorithms.
  *
  */
-template<class T, unsigned D, bool ORTHO = false>
-struct CrystalLattice
+template<class T, unsigned D>
+struct CrystalLattice: public LRBreakupParameters<T, D>
```

Comment:
The class comment seems out of date.  Please avoid changing code without changing the explanatory comments. The indices comment is cryptic.  It looks like this facility is being removed from the code.

Reply:
I removed the indices comment.